service.py

```diff
@@ -0,0 +1,47 @@
+class FacultyService:
+    # In-memory storage for demonstration
+    # Each faculty: {"id": int, "name": str, "department": str, "title": str}
+    _faculties = []
```

Comment:
this list should be list of models of faculty. Firstly create Model Faculty with all properity it needs

Reply:
Ignore this if useing db